syntax = "proto3";
package dataheap2;

message HistoryRequest {
    int64 start_time = 1;
    int64 end_time = 2;
    int64 interval_ns = 3;
}

message HistoryResponse {
    string metric = 1;
    repeated int64 time_delta = 2;
    repeated double value_min = 3;
    repeated double value_max = 4;
    repeated double value_avg = 5;
}
